meta-balena-common/recipes-kernel/linux/kernel-image-initramfs.bb

```diff
@@ -10,6 +10,9 @@ do_install() {
     mkdir -p ${D}/boot
     for type in ${KERNEL_IMAGETYPE}; do
         install -m 0644 ${DEPLOY_DIR_IMAGE}/${type}-initramfs-${MACHINE}.bin ${D}/boot/${type}
+        if [ -f "${DEPLOY_DIR_IMAGE}/${type}.sig" ]; then
+            install -m 0644 ${DEPLOY_DIR_IMAGE}/${type}.sig ${D}/boot/${type}.sig
```

Comment:
The two lines above should use `${DEPLOY_DIR_IMAGE}/${type}.initramfs.sig`

Reply:
Done @mtoman